utils/draw_box_on_image.py

```diff
@@ -3,6 +3,35 @@
 import matplotlib.pyplot as plt
 
 def draw_line(image, y1, x1, y2, x2, line_type):
+    ''' Helper function to draw lines on an image. 
+  
+    Parameters
+    ----------
+    images: np.array
+        The correponding image.
+
+    y1: int
+        Starting y position of the line
+
+    x1: int
+        Starting x position of the line
+
+    y2: int
+        Ending y position of the line
+
+    x2: int
+        Ending x position of the line
+
+    line_type: str
+        Options of the line_type is dotted or full.
```

Comment:
please list options in doc string

Reply:
the options are "dotted" or "full"

Reply:
👍 please add quotes in the doc string then. Or list like: `Type of line: ["dotted", "full"]`